.travis.yaml

```diff
@@ -0,0 +1,11 @@
+language: python
+python: 2.7, 3.5
+env:
+  - TOXENV=flake8
+install:
+  - pip install tox
+script:
+  - tox
+notifications:
+email: rpappalax@gmail.com 
```

Comment:
w/ chartjes taking over the project, should notifications be tweaked to represent this?